conda-forge.yml

```
@@ -1,3 +1,5 @@
+bot:
+  inspection: update-grayskull
 build_platform:
   linux_aarch64: linux_64
   linux_ppc64le: linux_ppc64le
```

Am curious if we would be willing to try cross-compiling for `linux_ppc64le` since Travis appears to be having issues and we already do this for `linux_aarch64`

Please let me know what you think 🙂 

```suggestion
  linux_ppc64le: linux_64
```

---

Worth a try again, it's been a few years. It may run into https://github.com/PyWavelets/pywt/issues/607 again, but then again it may not.

---

Alternatives would be

* Stick with Travis and babysit CI (requires manual restarts)
* Disable `linux_ppc64le` altogether